src/containers/LandingContainer.jsx

```diff
@@ -103,6 +104,20 @@ class InvestigationsLanding extends React.PureComponent {
                   </div>
                 </div>
               </>
+            ) : (
+              !fresh && (
+                <div className="space-bottom">
+                  <Button
+                    flat
+                    primary
+                    swapTheming
+                    onClick={this.dispatch.empty}
+                    style={{ backgroundColor: '#df0039' }}
+                  >
+                    <Trans>interface::actions.reset</Trans>
```

Comment:
this is just a simple "Reset", but perhaps some language around it would be helpful?